test/UnitTest/Microsoft.AspNet.OData.Test.Shared/EnableNestedPathsTest.cs

```diff
@@ -223,7 +223,7 @@ public async Task EnableNestedPaths_Returns404_WhenPathHasUnsupportedSegments(st
             Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
         }
 
-        [Fact]
```

Comment:
why ship this test case?

Reply:
Test Not relevant anymore as it works differently now and new test cases are added. Removed